extensions/eclipse-che-theia-workspace/src/node/che-workspace-server.ts

```diff
@@ -56,6 +56,11 @@ export class CheWorkspaceServer extends DefaultWorkspaceServer {
 }
 
 function isMultiRoot(workspace: Workspace): boolean {
-  const devfile = workspace.devfile;
-  return !!devfile && !!devfile.attributes && !!devfile.attributes.multiRoot && devfile.attributes.multiRoot === 'on';
+  const devWorkspaceNamespace = process.env.DEVWORKSPACE_NAMESPACE;
```

Comment:
we created abstraction on how workspaces are retrieved (no process.env in the code except in remote-api-services)

so if you need to know, probably you can add a method in the api

another hint is to check if you've runtime in workspace (undefined in the case of DevWorkspace)

Reply:
else, we could always provide the attribute `multiRoot` in case of DevFile v2 implementation (so we don't change the consumer code)

Reply:
Thank you, Florent, for the ideas how to handle this problem.

AFAIK we are going to turn on `multi-root` by default very soon.
We will consider that:
- the `multi-root` is `OFF` **only** if there is the attribute `multiRoot` with `off` value
- the `multi-root` is `ON` for other cases

So, the check will be like:
```
function isMultiRoot(workspace: Workspace): boolean {
  return workspace.devfile?.attributes?.multiRoot !== 'off';
}
```
Taking into account the info above I think we should apply a solution from your first comment, so:
- adding a method to the API 
or 
- check if you've runtime in workspace object (undefined in the case of DevWorkspace)

If I didn't missed something - I guess the second one (runtime in workspace object) - is the simplest way.

@benoitf @azatsarynnyy 
WDYT?

Reply:
yes go the simplest (2nd case as it's temporary)